images/build/Dockerfiles/Dockerfile

```diff
@@ -235,12 +211,6 @@ RUN cp ${IMAGES_DIR}/build/dotNetCoreAll.Readme.md /opt/dotnet/Readme.md
 # Copy NodeJs, NPM and Yarn related content
 COPY --from=node-install /opt /opt
 
-# Install PHP pre-reqs
-RUN ${IMAGES_DIR}/build/php/prereqs/installPrereqs.sh
```

Comment:
These are just the pre-reqs which I think are required to be present in the build image. No?